packages/vendor-claude-code/src/node.ts

```diff
@@ -1,7 +1,18 @@
+import { getLogger, isVSCodeEnvironment } from "@getpochi/common";
 import { registerVendor } from "@getpochi/common/vendor";
+import { initializeProxy } from "./proxy";
+import { VendorId } from "./types";
 import { ClaudeCode } from "./vendor";
 
+const logger = getLogger(`${VendorId}-node`);
+
 registerVendor(new ClaudeCode());
 
+if (isVSCodeEnvironment()) {
```

Comment:
create isVSCodeEnvironment in this package directly, do not reuse the code from common package